extensions/interactions/ItemSelectionInput/ItemSelectionInput.html

```diff
@@ -1,13 +1,76 @@
+<style>
+  /* The following is needed so that the radio button does not show */
+  input[type="radio"] {
```

Comment:
@BenHenning , @kevinlee12 : This is the reason for the failing protractor tests. The radio buttons are simply not showing on the protractor page, so there's nothing too click. This CSS rule is too broad.

Isn't the radio button hidden anyway due to the ng-switch-when?